mitodl/open-discussions
# generate rss for all MIT podcast episodes

#### Pre-Flight checklist

#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/3047

#### What's this PR do?
This pr updates the podcasts pipeline to generate an rss file with all MIT episodes.
It also creates a view to display the rss file at /podcasts/rss_feed

#### How should this be manually tested?
run `docker-compose run web ./manage.py backpopulate_podcast_data`
A rss file should be created in the `/podcast_rss directory`
http://localhost:8063/podcasts/rss_feed should display the rss feed
